sceptre/scipool/config/bmgfki/sc-product-assoc-ec2-linux-windows-jumpcloud.yaml

```diff
@@ -0,0 +1,10 @@
+template_path: "sc-ec2-action-associations.yaml"
```

Comment:
The name suggests that this product is accessed through JumpCloud.  Is that what we want for an S.C. instance set up for external users?